packages/quantic/force-app/main/default/.eslintrc.json

```diff
@@ -20,6 +20,13 @@
       "rules": {
         "@lwc/lwc/no-async-operation": "off"
       }
+    },
+    {
+      "files": ["**/e2e/*.ts"],
+      "parser": "@typescript-eslint/parser",
+      "rules": {
+        "no-redeclare": ["error", {"builtinGlobals": false}]
```

Comment:
it's because Jest defines `expect` as a global variable, and in the e2e tests we importing `expect` from playwright,
so eslint  creates an error saying that we can't re-declare a global variable.

this rule avoids this error.
An alternative solution would be to do:
```
import { expect as testExpect } from '@playwright/test';
```
by changing the alias of the imported expect playwright.